### ISSUE TITLE
Bug in Experiment Tool: error message in Create Instance when creating an already existing one

### ISSUE THREAD

**Issue by prudhomm** [MSO4SC/MSOPortal#94]
This is a case of Bad UX
/cc @emepetres @victorsndvg @Trophime @gdolle 

here is the screenshot.
<img width="688" alt="screenshot 2018-06-07 22 52 14" src="https://user-images.githubusercontent.com/151693/41125992-60bf7c40-6aa6-11e8-9368-1cfade06f525.png">

**Comment by Trophime** [comment]
for that kind of here, a login/logout may help.
Working in "incognito" windowns may also help. It's a way to start with  empty caches...

But I agree: this is confusing for the end user, "to say the least"

**Comment by emepetres** [comment]
Closely related to #93 I think.

**Comment by prudhomm** [comment]
Indeed, it might be. Although the way to get it is slightly different
